src/analysis/retail/rogue/shared/index.ts

```diff
@@ -5,7 +5,7 @@ export { default as EnergyDetails } from './EnergyDetails';
 export { default as EnergyTracker } from './EnergyTracker';
 export { default as FilteredDamageTracker } from './FilteredDamageTracker';
 export { default as FinisherTracker } from './FinisherTracker';
-export { default as InvigoratingShadowdust } from '../subtlety/modules/talents/InvigoratingShadowdust';
+export { default as InvigoratingShadowdust } from './shadowlands/legendaries/InvigoratingShadowdust';
```

Comment:
This doesn't quite seem like the right import path. If the intent was to make it so that the other specs could continue to have the benefit of the legendary, then it's fine.

Reply:
@ToppleTheNun yes, I wanted to support the legendary here.